_assets/ci/Jenkinsfile.tests-rpc

```diff
@@ -42,11 +58,11 @@ pipeline {
     always {
       script {
         archiveArtifacts(
-          artifacts: '**/results.xml',
+          artifacts: 'integration-tests/reports/*.xml, integration-tests/*.log, integration-tests/coverage/coverage.html',
```

Comment:
If this is a list we should indicate it as such:
```suggestion
          artifacts: ['integration-tests/reports/*.xml, integration-tests/*.log, integration-tests/coverage/coverage.html'],
```

Reply:
Hmm, @jakubgs I'm confused actually.

Jenkins docs [say](https://www.jenkins.io/doc/pipeline/steps/core/) that `artifacts` is of string type:

<img width="585" alt="image" src="https://github.com/user-attachments/assets/0cf2f650-b111-4669-9682-60f7d29bb81b">

And show this as example to pass multiple artifcats:
> How to archive multiple artifacts with different patterns:
> `archiveArtifacts artifacts: 'target/*.jar, target/*.war'`